mule-extensions-api/src/main/java/org/mule/runtime/extension/api/model/error/ErrorModelBuilder.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.extension.api.model.error;
+
+import org.mule.runtime.api.meta.model.error.ErrorModel;
+import org.mule.runtime.api.meta.model.ModelProperty;
+
+import java.util.HashSet;
+
+/**
+ * Builder pattern implementation to build {@link ErrorModel} instances.
+ *
+ * @since 1.0
+ */
+public final class ErrorModelBuilder {
```

Comment:
this relates to the comment in the other PR. If you chose the builder path, then the builder should also be part of the mule-api, following the pattern of the LayoutModel. 

Again, ErrorModel is something that I think will grow. I wouldn't be surprised if it ends up becoming a ParmeterizedModel. I don't think this is the way to go